internal/tunnel/tunnel.go

```diff
@@ -23,17 +23,17 @@ type Tunnel struct {
 
 type Tunneler interface {
 	// StartTunnel connects to load balancer tunnel port and creates connection
-	StartTunnel(nodeID string, tunnelURL string, token string)
+	StartTunnel(nodeID string, tunnelURL string, token string, port int)
 }
 
-func (t *Tunnel) StartTunnel(nodeID string, tunnelURL string, token string) {
+func (t *Tunnel) StartTunnel(nodeID string, tunnelURL string, token string, port int) {
 	c, err := client.NewClient(&client.ClientConfig{
 		ServerAddress: tunnelURL,
 		Tunnels: map[string]*client.Tunnel{
 			"default": {
 				Protocol:   Protocol,
 				Addr:       t.NodeRPCURL.Host,
-				RemoteAddr: RemoteAddr,
+				RemoteAddr: "0.0.0.0:" + strconv.Itoa(port),
```

Comment:
eh?

Reply:
Well string(port) returns weird signs so this is the way to convert it string
Port is port that is returned in response that we send to assign as we can't auto assign and save to database
In vedran I will send PR with that change
So tunnelURL is url that we should connect to and port is the port we should send to be assigned on load balancer

Reply:
but ain't this suppose to be load balancer IP + tunnel port?